pkg/controller/healthz.go

```diff
@@ -53,7 +56,9 @@ func HandleHealthz(db *database.Database, h render.Renderer) http.Handler {
 				return
 			}
 		case "alerts":
-			// TODO(ych): fire a metric and configure an alert
+			// Alerts increments a metric that fires an alert after some interval. It
```

Comment:
I'm not convinced we should do this - because those endpoints are currently accessible to the public through the load balancer.

Anyone can page us at will...

Reply:
Removed this and updated the OR to reflect the correct description.